odl/operator/operator.py

```diff
@@ -2055,12 +2055,19 @@ def derivative(self, x):
     def adjoint(self):
         """Adjoint of this operator.
 
-        The adjoint of the operator vector multiplication is the
-        vector multiplication of the operator adjoint:
+        For real spaces, the adjoint of the operator left vector multiplication
+        is the right multiplication of the given operator by the given vector.
 
             ``OperatorLeftVectorMult(op, y).adjoint ==
             OperatorRightVectorMult(op.adjoint, y)``
 
+        For complex spaces, the adjoint of the operator left vector
```

Comment:
Not quite sure we need two different comments here, this reduces to the above case. The only reason we have an if statement is that conj() produces a copy, which is bad for performance.

I'd try to "merge" these so to say, only leaving the code for the complex case but perhaps mentioning that the conj does nothing for real vectors.